scripts/ansible/roles/linux/openenclave/tasks/validation.yml

```diff
@@ -37,9 +37,9 @@
   register: clang_check_version
   failed_when: clang_check_version.stdout.find(clang_target_version) == -1
 
-- name: Ocaml version check
-  shell: "ocaml -version"
+- name: Esy version check
```

Comment:
Should this continue to check for `ocaml` since tools/oeedger8r/CMakeLists.txt still calls directly into `ocamlopt` rather than esy? (i.e. is the implication that esy presence will always equate to ocaml presence moving forwards?)

Reply:
No, as this is really just part of #2221 and was an attempt to solve a chicken/egg problem with updating dependencies. I am going to close this PR as John and I are working to solve updating the CI machines outside of this.